pointerevents/pointerevent_support.js

```diff
@@ -12,10 +12,12 @@ var All_Pointer_Events = [
 
 // Check for conformance to PointerEvent interface
 // TA: 1.1, 1.2, 1.6, 1.7, 1.8, 1.9, 1.10, 1.11, 1.12, 1.13
-function check_PointerEvent(event) {
-    var pointerTestName = event.pointerType + ' ' + event.type;
+function check_PointerEvent(event, testNamePrefix, pointerEventInterface) {
+    if (testNamePrefix === undefined)
+        testNamePrefix = "";
+    var pointerTestName = testNamePrefix + ' ' + event.pointerType + ' ' + event.type;
     test(function () {
-        assert_true(event instanceof PointerEvent, "event is a PointerEvent event");
+        assert_true(event instanceof (pointerEventInterface || PointerEvent), "event is a PointerEvent event");
```

Comment:
Rather than pass this in, could you just check `event instanceof event.target.ownerDocument.abstractView.PointerEvent`?  Then that would validate that the interface belongs to the same window as the target - a nice extra property to verify explicitly.